tool_helper_utils/__init__.py

```diff
@@ -2,12 +2,30 @@
 import sys
 
 
+class AbstractBaseFactoryFactory:
+    def build():
```

Comment:
This needs to be annotated with `abc.abstractmethod`. Also the method body should just be the ellipsis (`...`).